app/controllers/concerns/curation_concerns/curation_concern_controller.rb

```diff
@@ -46,10 +45,22 @@ def create
   # @raises CanCan::AccessDenied if the document is not found
   #   or the user doesn't have access to it.
   def show
-    _, document_list = search_results(params, CatalogController.search_params_logic + [:find_one])
-    curation_concern = document_list.first
-    raise CanCan::AccessDenied.new(nil, :show) unless curation_concern
-    @presenter = show_presenter.new(curation_concern, current_ability)
+    respond_to do |wants|
+      wants.html do
+        _, document_list = search_results(params, CatalogController.search_params_logic + [:find_one])
+        curation_concern = document_list.first
+        raise CanCan::AccessDenied.new(nil, :show) unless curation_concern
+        @presenter = show_presenter.new(curation_concern, current_ability)
+      end
+      wants.json do
+        # load and authorize @curation_concern manually
+        # This has to use #find instead of #load_instance_from_solr because
+        # we want to return values like generic_file_ids in the json
+        @curation_concern = curation_concern_type.find(params[:id]) unless curation_concern
+        authorize! :show, @curation_concern
```

Comment:
It's skiped on show, see https://github.com/projecthydra-labs/curation_concerns/blob/json_api/app/controllers/concerns/curation_concerns/curation_concern_controller.rb#L19 Because we want to load it from solr (for html) and not from Fedora.